azure-pipelines.yml

```diff
@@ -200,7 +203,7 @@ stages:
             inputs:
               mavenPomFile: 'pom.xml'
               goals: 'test'
-              options: $(MVN_OPTS_TEST) -Punit-tests -pl $(JOB4_UT_MODULES)
+              options: $(MVN_OPTS_TEST) -Punit-tests -pl $(JOB4_UT_MODULES) -DfailIfNoTests=false -DwildcardSuites="abc" -Dtest=TestHoodieDeltaStreamer#testForceEmptyMetaSync
```

Comment:
I'm assuming this is just for debugging purposes. Do we still need changes in this file?

Reply:
Yes, this is for debugging purposes only.  Reverted the changes.